__tests__/app.test.js

```diff
@@ -148,6 +148,207 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200 returns an array of articles filtered by [mitch], default sort (created_at) and default order (DESC)", () => {
+    const desiredArticle = {
+      author: expect.any(String),
+      title: expect.any(String),
+      article_id: expect.any(Number),
+      topic: "mitch",
+      created_at: expect.any(String),
+      votes: expect.any(Number),
+      article_img_url: expect.any(String),
+      comment_count: expect.any(String),
+    };
+
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(12);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+        articles.forEach((article) => {
+          expect(article).toMatchObject(desiredArticle);
+        });
+      });
+  });
+  test("200 returns an array of articles filtered by [mitch], default sort (created_at) and order [ASC]", () => {
+    const desiredArticle = {
+      author: expect.any(String),
+      title: expect.any(String),
+      article_id: expect.any(Number),
+      topic: "mitch",
+      created_at: expect.any(String),
+      votes: expect.any(Number),
+      article_img_url: expect.any(String),
+      comment_count: expect.any(String),
+    };
+
+    return request(app)
+      .get("/api/articles?topic=mitch&order=ASC")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(12);
+        expect(articles).toBeSortedBy("created_at");
+        articles.forEach((article) => {
+          expect(article).toMatchObject(desiredArticle);
+        });
+      });
+  });
+  test("200 returns an array of articles filtered by [cats], default sort (created_at) and default order (DESC)", () => {
+    const desiredArticle = {
+      author: expect.any(String),
+      title: expect.any(String),
+      article_id: expect.any(Number),
+      topic: "cats",
+      created_at: expect.any(String),
+      votes: expect.any(Number),
+      article_img_url: expect.any(String),
+      comment_count: expect.any(String),
+    };
```

Comment:
Since you're not mutating this at all it would be acceptable to declare it outside the test and reuse that variable